client/src/featureform/dashboard_metadata.py

```diff
@@ -96,59 +96,59 @@ def training_set_variant(variantData):
 
     return variantDict, allVariantList, variants
     
-def getTrainingSetFeatures(featureList):
+def getTrainingSetFeatures(feature_list):
     # parse the featureList str(list of tuples)
     featureVariantTuple = []
-    features = [tuple(featureTuple.split('\', \'')) for featureTuple in re.findall("\(\'(.*?)\'\)", featureList)]
+    features = [tuple(feature_tuple.split('\', \'')) for feature_tuple in re.findall("\(\'(.*?)\'\)", feature_list)]
     # Iterate throigh the list and get one large tuple of tuples
     for feature in features:
-        featureVariantTuple += sqlObject.getNameVariant("feature_variant", "featureName", feature[0], "variantName", feature[1])
+        featureVariantTuple += sql_object.get_name_variant("feature_variant", "featureName", feature[0], "variantName", feature[1])
     
     return featureVariantTuple
 
-def training_sets(rowData):
-    variantData = training_set_variant(sqlObject.getVariantResource("training_set_variant", "trainingSetName", rowData['name']))
+def training_sets(row_data):
+    variant_data = training_set_variant(sql_object.get_variant_resource("training_set_variant", "trainingSetName", row_data['name']))
     return TrainingSetResource( 
-                rowData['type'], #type
-                rowData['defaultVariant'], #defaultvariant
-                rowData['name'], #name
-                variantData[0], #variantsDict
-                variantData[1] #all variants
+                row_data['type'], #type
+                row_data['defaultVariant'], #defaultvariant
```

Comment:
Should make dictionary keys lowercase as well

Reply:
Ya sorry, I'll ask Ksshiraja to do the rest, there were too many. I just wanted to make sure I didn't break anything before I hand it over so I open a draft PR